### PR TITLE
Feature/v2.0.0

### PR DESCRIPTION
**Pull request by erik-dunteman**
# This is our first major breaking change

# Why:
- We needed to add a strategy argument, to allow batchSize parallelization across many inference servers. As part of this, we needed to be able to return a list of outputs rather than a single output, breaking backwards compatibility 
- We needed compatibility to the new "pull" style inference servers we've implemented on the backend
- Our outputs were getting pretty frankensteiny across a wide range of use cases, so we're following a standard format without any tricky client parsing logic

# Testing:
Tests pass end to end for:
- New "pull" style inference servers
- Legacy "push" style inference servers

We tried our best to make all models still work through this new sdk. The API itself is a breaking change out of necessity, but if your inference call are breaking, please contact us.

### CODE REVIEW

File: banana_dev.egg-info/PKG-INFO [hunk 1]
```diff
@@ -1,6 +1,6 @@
 Metadata-Version: 2.1
 Name: banana-dev
-Version: 0.1.0
+Version: 1.0.0
```
Comment (kylejmorris) [inline banana_dev.egg-info/PKG-INFO:3]: in ur PR title you said v2.0.0 so is the 1.0.0 intentional here?

File: banana_dev/generics.py [hunk 2]
```diff
@@ -18,90 +18,85 @@
...
-            if "output" in dict_out['data']['taskOut']:
-                return dict_out['data']['taskOut']["output"]
-            else:
-                return dict_out['data']['taskOut']
-def start_main(api_key, model_key, model_parameters):
-    task_id = call_start_api(api_key, model_key, model_parameters)
-    return task_id
-def check_main(api_key, task_id):
-    dict_out = call_check_api(api_key, task_id)
+        dict_out = check_api(api_key, call_id)
```
Comment (kylejmorris) [inline banana_dev/generics.py:24]: so much cleaner, nice!

**Review by kylejmorris — APPROVED** [review]
lgtm overall with caveat of version #

File: banana_dev.egg-info/PKG-INFO (hunk 1 above)
Comment (erik-dunteman) [reply banana_dev.egg-info/PKG-INFO:3]: ohhh good catch

**Pull request merged**